### DISCUSSION TITLE
Welcome to FullPageOS Discussions!

### DISCUSSION THREAD

**Discussion by guysoft** [guysoft/FullPageOS#513] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to manage stuff that is not an issue directly with FullPageOS, configuration ideas, meta talks, etc.
   * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together ❤️

**Comment by masteroftherings** [comment]


**Reply by guysoft** [reply]
Hey, I suggest you post this in a separate discussion and not in the welcome message.
